js/memory.js

```diff
@@ -1,9 +1,40 @@
 class MemoryGame {
-  constructor(cards){
+  constructor(cards) {
     this.cards = cards;
-    // add the rest of the class properties here
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
   }
-  shuffleCards() {}
-  checkIfPair(card1, card2) {}
-  isFinished() {}
-}
\ No newline at end of file
+  shuffleCards() {
+    let j, x, i;
```

Comment:
Could the variables "j" and "x" be named more expressively? What do these variables represent, or contain?